deploy/operator.antrea.vmware.com_v1_antreainstall_cr.yaml

```diff
@@ -128,3 +128,4 @@ spec:
     # antrea-controller container.
     #selfSignedCert: true
   antreaImage: antrea/antrea-ubi:v0.9.1
+  antreaPlatform: kubernetes/openshift
```

Comment:
@ruicao93 @heypnus any suggestion about the default value of platform?

Reply:
If this operator has been released for OpenShift I think it should be openshift, but it didn't, right? Then both are ok for me and my preference is kubernetes.

Reply:
Thought it again, I think we can keep it "openshift" for now because the code for Kubernetes is not ready. When we really support both platforms, we will need separate folders, e.g. `deploy/openshift4` and `deploy/kubernetes` to hold the yaml files, then we can use "openshift" in `deploy/openshift4/operator.antrea.vmware.com_v1_antreainstall_cr.yaml` and "kubernetes" in `deploy/kbuernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml`

Reply:
Thanks, set it to "openshift" for now.

Reply:
Agree with `openshift`